Source/common/SNTXPCControlInterface.h

```diff
@@ -46,19 +47,9 @@ typedef NS_ENUM(NSInteger, SNTRuleAddSource) {
 ///
 ///  Config ops
 ///
-- (void)setClientMode:(SNTClientMode)mode reply:(void (^)(void))reply;
-- (void)setFullSyncLastSuccess:(NSDate *)date reply:(void (^)(void))reply;
 - (void)setRuleSyncLastSuccess:(NSDate *)date reply:(void (^)(void))reply;
 - (void)setSyncTypeRequired:(SNTSyncType)syncType reply:(void (^)(void))reply;
-- (void)setAllowedPathRegex:(NSString *)pattern reply:(void (^)(void))reply;
-- (void)setBlockedPathRegex:(NSString *)pattern reply:(void (^)(void))reply;
-- (void)setBlockUSBMount:(BOOL)enabled reply:(void (^)(void))reply;
-- (void)setRemountUSBMode:(NSArray *)remountUSBMode reply:(void (^)(void))reply;
-- (void)setEnableBundles:(BOOL)bundlesEnabled reply:(void (^)(void))reply;
-- (void)setEnableTransitiveRules:(BOOL)enabled reply:(void (^)(void))reply;
-- (void)setEnableAllEventUpload:(BOOL)enabled reply:(void (^)(void))reply;
-- (void)setDisableUnknownEventUpload:(BOOL)enabled reply:(void (^)(void))reply;
-- (void)setOverrideFileAccessAction:(NSString *)action reply:(void (^)(void))reply;
+- (void)postflightResult:(SNTConfigBundle *)result reply:(void (^)(void))reply;
```

Comment:
Still not a fan of this name.

Maybe `updateSyncSettings`?

Reply:
The intention here was to be less generic, and have an explicit callsite for the daemon to perform any actions once postflight was complete. So I'd prefer a name that made this clear. Maybe `postflightComplete`?